pending/dancer_core_send_as.pod

```diff
@@ -0,0 +1,125 @@
+=encoding utf8
+
+=head2 send_as BEER => 'Christmas Cheer'
+
+One Frequently Asked Question that arose in 2015 from several
+keen Dancer2 users was 
```

Comment:
Perhaps add a paragraph about the background and the change in Dancer2 that clarified how serializers worked. I would also link to the article explaining this -> http://advent.perldancer.org/2014/11 (Under "Always or never").